packages/bpk-component-calendar/src/utils.js

```diff
@@ -21,21 +21,32 @@ import {
   calendarDaySpacing,
 } from 'bpk-tokens/tokens/base.es6';
 
-const remToPx = value => {
-  let parsed = null;
+const splitToken = value => {
+  const match = value.match(/(^[+-]?(?:\d*\.)?\d+)([a-z]+)/i);
+  if (!match) {
+    // TODO: improve this
+    throw new Error('Invalid token');
+  }
+  const [_, val, unit] = match; // eslint-disable-line no-unused-vars
+  return [parseFloat(val), unit];
+};
 
-  if (/rem$/.test(value)) {
-    parsed = parseFloat(value.replace(/rem/, '')) * 16;
+export const getCalendarGridWidth = (multiplier = 1) => {
+  const [sizeValue, sizeUnit] = splitToken(calendarDaySize);
```

Comment:
I'm wondering if we should move this outside the function as this is a static value and will never change. It would also fail as soon as the JS file loads if the tokens are not correct.